planner/core/physical_plan_test.go

```diff
@@ -1550,6 +1604,20 @@ func (s *testPlanSuite) TestIndexJoinHint(c *C) {
 	}
 }
 
+func (s *testPlanSuite) TestTopNPushDownEmpty(c *C) {
+	store, dom, err := newStoreWithBootstrap()
```

Comment:
why move this test?

Reply:
because the test was between `TestDAGPlanBuilderWindow` and `doTestDAGPlanBuilderWindow`, so it might be better to move it out here.